### ISSUE TITLE
Training for classification is not learning anything.

### ISSUE THREAD

**Issue by virtuoso108** [MzeroMiko/VMamba#395]
For classification training on imagenet data with vmamba-tiny model the loss is 6.9 and does not get lower even through I keep training.
I changed the learning rate but that does not work. Could you please tell me how I can fix this problem?
